scripts/get_cmake_vars/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ foreach(_lang IN LISTS VCPKG_LANGUAGES)
     list(APPEND VCPKG_DEFAULT_VARS_TO_CHECK CMAKE_${_lang}_COMPILE_FEATURES)
     list(APPEND VCPKG_DEFAULT_VARS_TO_CHECK CMAKE_${_lang}_EXTENSION)
     # Probably never required since implicit. 
-    #list(APPEND VCPKG_DEFAULT_VARS_TO_CHECK CMAKE_${_lang}_IMPLICIT_LINK_FRAMEWORK_DIRECTORIES)
+    list(APPEND VCPKG_DEFAULT_VARS_TO_CHECK CMAKE_${_lang}_IMPLICIT_LINK_FRAMEWORK_DIRECTORIES)

```

Comment:
Why change this?

Reply:
This is the variable of interest, and it is not made available by default.
To avoid the change to the script, it may be possible to pass the variable name via `OPTIONS`. I can test this later.

Reply:
Done.

Reply:
Will test later.

Reply:
All right.
Basically, this PR has two contributions:
- Using that CMake variable.
- Using a more explicit search and replace.

You might want to use a the second even without the first. The question is if you want to add osx system details to vcpkg scripts, or leave it to CMake's system detection scripts.

In addition, I wonder if non-system frameworks need the `-framework` option, too.

Reply:
Works, but should we add prefix `\$<\$<PLATFORM_ID:Darwin>` ?

Reply:
> Not for angle:
- CMAKE_SYSTEM_FRAMEWORK_PATH has more options alternatives than CMAKE_CXX_IMPLICIT_LINK_FRAMEWORK_DIRECTORIES, and it is independent of language. 
  See my initial post.
- Leading white space? Interesting... 

I'm not sure if I will try angle soon on my entry-level MBA...

Reply:
> \$<\$<PLATFORM_ID:Darwin>

Will these configuration files ever slip out of the Darwin triplet?

Reply:
It only exists in Darwin, I can't export this file from my osx machine, do you need other informations?

Reply:
> Not for angle:

Problem solved.